modules/RepConvGRC.js

```diff
@@ -1731,14 +1735,16 @@ function _RepConvGRC() {
         var website;
         // if (RepConv.active.statsGRCL == 'grepostats') {
         //     website = 'http://' + Game.market_id + '.grepostats.com/world/' + Game.world_id + '/'+what+'/' + who_id;
-        // } else 
+        // } else
         if (RepConv.active.statsGRCL == 'grepointel') {
             if (what == 'player') {
                 what = 'pn';
             } else if (what == 'alliance') {
                 what = 'an'
             }
             website = 'http://grepointel.com/track.php?server='+Game.world_id+'&'+what+'='+who_name+'&rt=overview';
+        } else if (RepConv.active.statsGRCL == 'grepolife' && (["ru", "us", "de", "en", "fr", "es", "br", "pl", "nl", "it", "pt", "ro"].indexOf(Game.world_id.slice(0, 2)) >= 0)) {
```

Comment:
What about the special worlds, e.g. ru9 and ru17?

Reply:
Fixed